.gitignore

```diff
@@ -0,0 +1 @@
+**target/
```

Comment:
why `**`?

Reply:
Ignore all target directories in this workspace.